app/views/govuk_publishing_components/components/_accordion.html.erb

```diff
@@ -14,6 +14,11 @@
   data_attributes ||= {}
   data_attributes[:module] = 'gem-accordion'
   data_attributes[:anchor_navigation] = anchor_navigation
+  data_attributes[:show_text] = t("components.accordion.show")
+  data_attributes[:hide_text] = t("components.accordion.hide")
+  data_attributes[:show_all_text] = t("components.accordion.show_all")
+  data_attributes[:hide_all_text] = t("components.accordion.hide_all")
+  data_attributes[:this_section_visually_hidden] = t("components.accordion.this_section_visually_hidden")
```

Comment:
The i18n `t` method will fall back to the default language when a key isn't present in the current language - so that could mean that a page in Spanish with the accordion might have the controls in English if there's no translations available. 

This mixing of languages is fine, as long as the `lang` attribute is used. Is this something that should be added?

Reply:
Good point. I'll drop in some `lang` magic to account for this.

Reply:
That works for now, since the translation is only available in English. There's now a risk that someone could now add translations for another language without realising that the `lang` attribute is hardcoded though.

Reply:
Since https://github.com/alphagov/govuk_publishing_components/pull/1992 was merged in and released, I think it might be worth thinking about how we can get this working without hardcoding the `lang` attribute. Happy to pair on this if you'd like?

Reply:
Agreed. Lemme have a crack at it and I'll get in touch with you if I run into problems.

Reply:
Updated. Lemme know your thonks.

Reply:
Following a slack chat I've now updated this again to instead use `t_locale` plus some fancy data attribute management. Lemme know what you think.